source/develop/contribute.rst

```diff
@@ -0,0 +1,27 @@
+Contribute
+**********
+
+Contributions of all kinds are welcome and encouraged.
+
+To contribute code or documentation, please open a `pull request <https://github.com/LinuxForHealth/connect/pulls>`_.
+
```

Comment:
How about "To contribute code or documentation for an existing issue, please open a pull request"?  Then for new features and issues, you mention opening an issue first, which is perfect.